src/language/variables.md

```diff
@@ -38,6 +38,34 @@ String name = 'Bob';
   of using `var`, rather than type annotations, for local variables.
 {{site.alert.end}}
 
+## Null safety
+
+The Dart language enforces sound null safety.
+
+Null safety prevents errors that result from unintentional access
+of variables set to `null`.
+
+For example, if a method expects an integer but receives `null`,
+your app causes a runtime error.
```

Comment:
Thanks @munificent for this clarification. I went with a little of both:

```markdown
For example,
say you want to find the absolute value of a variable `i` of type `int`.
If `i` receives `null`, `i.abs()` causes a null dereference error.

In other languages, if you allow method calls and property accesses on
nullable expressions, this could lead to runtime errors.
As the Dart compiler prohibits these actions, its apps can't cause
these runtime errors.
```

Reply:
I wouldn't say "If `i` receives `null`". In an object-oriented language "receive" is usually used for method calls ("message sends" in Smalltalk parlance). I'd just say "If `i` is `null`".